src/components/common/Input/Input.tsx

```diff
@@ -0,0 +1,30 @@
+import classNames from 'classnames';
+import React, { FC } from 'react';
+import { InjectedIntlProps, injectIntl } from 'react-intl';
+import { CustomInput as BTInput, CustomInputProps as BTInputProps } from 'reactstrap';
+
+import './Input.scss';
+
+export type InputProps = BTInputProps & InjectedIntlProps;
+
+export const Input: FC<InputProps> = ({
+  className,
+  type,
+  placeholder,
+  label,
+  intl: { formatMessage },
+  ...rest
+}) => {
+  return (
+    <BTInput
+      id="vene-custom-input-field"
+      className={classNames('vene-input', className)}
+      type={type}
+      label={label ? formatMessage({ id: label as string }) : ''}
+      placeholder={placeholder ? formatMessage({ id: placeholder }) : ''}
+      {...rest}
+    />
+  );
+};
+
+export default injectIntl(Input);
```

Comment:
We have some form fields components under `src/components/forms/fields/`, how about keeping all form fields in that folder for the consistency sake?